TAF/utils/scripts/docker/get-compose-file-perfermance.sh

```diff
@@ -10,10 +10,12 @@ USE_SECURITY=${2:--}
 [ "$USE_SECURITY" != '-security-' ] && USE_NO_SECURITY="-no-secty"
 
 # # sync docker-compose file from developer-script repo
-./sync-nightly-build.sh master ${USE_NO_SECURITY} ${USE_ARM64}
+./sync-nightly-build.sh master "${USE_NO_SECURITY}" "${USE_ARM64}" "-perf"
 
-cp docker-compose-nexus${USE_NO_SECURITY}${USE_ARM64}.yml docker-compose.yaml
+cp docker-compose-taf-perf-nexus${USE_NO_SECURITY}${USE_ARM64}.yml docker-compose-end-mqtt.yaml
```

Comment:
Test script use docker-compose-mqtt.yaml, please see [deploy-edgex.sh](https://github.com/edgexfoundry/edgex-taf/blob/a95419d5f7fb52d948eb99aa833bebb094c57a49/TAF/utils/scripts/docker/deploy-edgex.sh#L1-L17) and [robot file](https://github.com/edgexfoundry/edgex-taf/blob/a95419d5f7fb52d948eb99aa833bebb094c57a49/TAF/testScenarios/performanceTest/performance-metrics-collection/5_event_exported_time/event_exported_time.robot#L15). 

Also need to have docker-compose.yaml file without app-service.

Reply:
fixed. Thanks!

Reply:
@lenny-intel The tests need to have docker-compose.yaml without app-service-mqtt-export and mqtt-broker.

Reply:
fixed.